article/app/model/dotcomponents/DotcomponentsDataModel.scala

```diff
@@ -501,7 +502,8 @@ object DotcomponentsDataModel {
         articlePage.related.hasStoryPackage,
         article.content.showInRelated,
         article.trail.isCommentable,
-        linkedData
+        linkedData,
+        ContentLayout.showBottomSocialButtons(article)
```

Comment:
Does this apply to share buttons just at the bottom or everywhere?

Reply:
Just at the bottom

Reply:
This is almost not relevant for DCR but we should apparently remove them in cases where the article is under 700 chars https://github.com/guardian/frontend/pull/21624/files#diff-09f6bf537e991ac8fae5857dbae0c6bbR12